oellm/template.sbatch

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#SBATCH --job-name=oellm-eval
+#SBATCH --time=00:30:00
+#SBATCH --gres=gpu:$NUM_GPU_PER_NODE
+#SBATCH --output=logs/%x-%A-%a.out # TODO 
+#SBATCH --partition=$DEFAULT_PARTITION
+#SBATCH --account=$EVAL_ACCOUNT
+#SBATCH --error=logs/%x-%A-%a.err # TODO
+#SBATCH --array=0-{array_limit}%{max_array_len}
+
+
+CSV_PATH="{csv_path}"
```

Comment:
Having an echo here which says key info such as what is going to be evaluated would be good here.